packages/openapi-typescript/test/transform/schema-object/object.test.ts

```diff
@@ -28,6 +28,27 @@ describe("transformSchemaObject > object", () => {
         // options: DEFAULT_OPTIONS,
       },
     ],
+    [
+      "basic > options.propertiesRequired: tre",
```

Comment:
nit: not particular about the naming of the test, but it should just describe it’s not _just_ about setting the flag to `true`; it’s about the behavior of `true` + the required array

```suggestion
      "options > propertiesRequired + required array",
```